packages/reports/addon/templates/components/dimension-selector.hbs

```diff
@@ -18,16 +18,17 @@
     {{#if item.dateTimeDimension}}
       {{#if (feature-flag "enableRequestPreview")}}
         <div class="grouped-list__item-container {{if this.selectedTimeGrain "grouped-list__item-container--selected grouped-list__item-container--disabled"}}">
-          <span class="grouped-list__item-label"
+          <button class="grouped-list__item-label"
             role="button"
```

Comment:
do we need role button if it's a button?

Reply:
From what I can tell, it doesn't hurt anything and there may be instances where accessibility software benefits from an explicit role rather than relying on the default.